src/app/components/home/home.component.spec.ts

```diff
@@ -60,17 +60,4 @@ describe('HomeComponent', () => {
   it('should be created', () => {
     expect(component).toBeTruthy();
   });
-
```

Comment:
I see why you have removed the test specs here. But add to **TODO's** to add new test cases here.

Reply:
done